shell/ci/release/docker.sh

```diff
@@ -139,24 +159,27 @@ build_and_push_image() {
       buildContext="$(get_repo_directory)/deployments/$image"
     fi
   fi
+  args+=("$buildContext")
+
+  # Check if the arguments include --push, if so, we'll push
```

Comment:
Can you set when you append the argument, instead of doing a loop a few lines below?

Reply:
I wanted to detach from the CIRCLE_TAG check in case we moved away from it, but yeah, that's reasonable.